### PR TITLE
feat: resize fonts in genie figure

### LINKED ISSUE #106: Resize GENIE Figure
Increase size of GENIE figure for better visualization

### PR DESCRIPTION
**Pull request by jarbesfeld**
Closes #106 

Resize GENIE graph and include updated graph on README.

### CODE REVIEW

**Comment by jarbesfeld** [comment]
Will resolve merge conflicts first

**Review by korikuzma — CHANGES_REQUESTED** [review]

**Review by korikuzma — APPROVED** [review]

**Comment by korikuzma** [comment]
@jarbesfeld there are conflicts. Can you resolve?

**Comment by jarbesfeld** [comment]
@korikuzma I realized I haven't resolved merge conflicts yet on the command line. Should I just follow the instructions here:
Screenshot 2024-01-15 at 9 29 10 AM

**Comment by korikuzma** [comment]
@jarbesfeld yes

**Comment by jarbesfeld** [comment]
@korikuzma Step 3 failed when I tried `git merge main`
rsjxa001@RESWCL4QK7DJL variation-normalizer-manuscript % git merge main
Auto-merging README.md
CONFLICT (content): Merge conflict in README.md
warning: Cannot merge binary files: analysis/genie/variant_analysis/genie_patient_matching.png (HEAD vs. main)
Auto-merging analysis/genie/variant_analysis/genie_patient_matching.png
CONFLICT (add/add): Merge conflict in analysis/genie/variant_analysis/genie_patient_matching.png
Auto-merging analysis/genie/variant_analysis/genie_search_analysis.ipynb
CONFLICT (content): Merge conflict in analysis/genie/variant_analysis/genie_search_analysis.ipynb
Automatic merge failed; fix conflicts and then commit the result.
Should I resolve the merge conflicts before this step?

**Comment by korikuzma** [comment]
@jarbesfeld yes

File: analysis/download_s3_files.ipynb
Comment (korikuzma) [inline analysis/download_s3_files.ipynb:1]: You shouldn't be making any changes to this. Revert this file to this commit ea62142 by doing the following:

`git checkout ea62142 analysis/download_s3_files.ipynb`

**Review by korikuzma — CHANGES_REQUESTED** [review]

**Review by wesleygoar — APPROVED** [review]

**Review by korikuzma — APPROVED** [review]

**Pull request merged**